test/lib/specHelperSpec.js

```diff
@@ -0,0 +1,111 @@
+import ActionPrototype from '../../src/lib/ActionPrototype'
```

Comment:
@BJK I still maintain that we need to organize the imports in some way, however minimal.